microsoft-edge/origin-trials/index.md

```diff
@@ -13,12 +13,73 @@ You can use Origin Trials to try out experimental APIs on your own live site for
 
 To support the experimental APIs, users don't need to go to `edge://flags` on their own device to access the experimental APIs that your site uses.  Instead, you register the origin that you want to use the experimental feature on (for example, `https://www.contoso.com:443`), and you'll then be sent a token that you add to the code of your website.
 
-To see a list of the available origin trials and register your origin to give them a try, see [Microsoft Edge Origin Trials](https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/).  To assess the suitability of using the experimental APIs on your website, check the minimum version requirements for Microsoft Edge, and the trial end date.
+To see a list of the available origin trials and register your origin to give them a try, see [Microsoft Edge Origin Trials](https://developer.microsoft.com/microsoft-edge/origin-trials).  To assess the suitability of using the experimental APIs on your website, check the minimum version requirements for Microsoft Edge, and the trial end date.
 
 You can also provide feedback to browser engineers and the web standards community about the design of the API, your use cases, or your experience using the APIs.
 
-For more information, see the **Getting Started** and **Developer Guidelines** sections in [Microsoft Edge Origin Trials](https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/).
+> [!IMPORTANT] 
+> If you're interested in the Ad Selection API origin trial, please register [here](https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/). [Learn more about sign-up.](../web-platform/ad-selection-api.md)
 
+## Getting Started
+
+To register for an origin trial, 
+1. [Go to Microsoft Edge Origin Trials](https://developer.microsoft.com/microsoft-edge/origin-trials) and select an active trial.
+
+2. Complete the registration form and click the **Submit** button to get a valid token. 
+
+To use an origin trial token, you can use one of these two options: 
+1. Using a `<meta>` Tag: Add an origin-trial tag to the head of any page. 
```

Comment:
change this numbered list to unordered list bc not sequential

i plan to push commit